src/components/MatPassContent.tsx

```
@@ -1185,39 +1533,39 @@ export default function MatPassContent() {
-                activeTab === 'acromatpass' 
-                  ? 'bg-primary text-white' 
-                  : 'bg-gray-100 text-gray-700 hover:bg-gray-200'
+                activeTab === "acromatpass"
+                  ? "bg-primary text-white"
+                  : "bg-gray-100 text-gray-700 hover:bg-gray-200"
               }`}
-              onClick={() => handleTabChange('acromatpass')}
+              onClick={() => handleTabChange("acromatpass")}
             >
               {language === "en" ? "Acro Yoga" : "Acro Yoga"}
             </button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)